### DISCUSSION TITLE
Printing a lens board

### DISCUSSION THREAD

**Discussion by chuckdries** [edgarkech/scalable_field_camera#15] [Q&A]
I noticed the parts list doesn’t mention the actual lens board itself. Would you recommend printing it or buying one? I found a couple different lens boards on thingiverse so people have done it before, but I worry a part that thin wouldn’t be totally light tight with PLA

**Comment by edgarkech** [comment]
I printed about 10 lens boards myself - black or dark grey with 100% infill was totally light tight. But keep in mind - especially if you are living in some warmer areas - PLA can get soft in the summer sun. So meanwhile I recommend printing stuff which may be exposed to temperatures of 40° C or higher (direct summer sun, inside a car in summer, ...) with PETG or some kind of other filament which is more temperature resistant than normal PLA.

**Reply by V-Roger** [reply]
I do use a printed lens board on my own scalable field camera, albeit only for the Fuji 135mm, the 90mm is way heavier and came with a "proper" board.

Never had any trouble with the printed one though! 100% infill in matte black did the trick.
